x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat_ecs/ml/linux_anomalous_network_activity.json

```diff
@@ -0,0 +1,51 @@
+{
+    "job_type": "anomaly_detector",
+    "description": "SIEM Auditbeat: Processes utilizing the network that doesn’t normally have network communications.",
```

Comment:
I think the `doesn't` should be `don't` for readability - as in `Processes utilizing the network that don't normally have network communications.`